docs/interop.ipynb

```diff
@@ -285,7 +285,7 @@
    "source": [
     "#### Gate conversion rules \n",
     "\n",
-    "Note: The standard Quantum Experience standard gates are defined in ([qelib.inc](https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/qasm/libs/qelib1.inc)) \n",
+    "Note: The standard Quantum Experience standard gates (defined in [qelib.inc](https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/qasm/libs/qelib1.inc)) are \n",
```

Comment:
We didn't touch this, but something feels off about the sentence `The standard Quantum Experience standard gates` due to redundant words.

Reply:
Done.